libindy/tests/anoncreds.rs

```diff
@@ -3695,4 +3695,32 @@ mod medium_cases {
             assert_code!(ErrorCode::CommonInvalidStructure, res);
         }
     }
+
+    //NOTE: This test is destructive to the common wallet, removing a credential used in other tests: it goes last
+    mod prover_delete_credential {
```

Comment:
I don't like an idea of tests based on running order.
I suggest leaving a test for `NotFoundError` only and moving deletion check to one of demo anoncreds tests or adding a new one.

Reply:
Moved to a new anoncreds demo test.